### DISCUSSION TITLE
Immich uploads locally

### DISCUSSION THREAD

**Discussion by Vansmak** [alexbelgium/hassio-addons#910] [Q&A]
I'm dumb when it comes to storage. I have a large enough ha drive that I can store all my photos in media/images on my ha. When I upload them with immich addon where are they uploaded to? I'd rather my media/images be that location.  Thanks for any help

**Comment by alexbelgium** [comment]
Hi, using the `data_location` option you can choose where you want your data stored. It also supports standard folders such as /media or /share that are on the HA base disk, or usage of external USB drives, or even smb. So all cases should be covered hopefully!
